test/models/archiver_test.rb

```diff
@@ -728,4 +728,39 @@ def create_api_key_with_webhook_for_perma_cc
     assert_equal ['perma_cc'], cached.keys
     assert_equal({ 'location' => 'http://perma.cc/perma-cc-guid-1'}, cached['perma_cc'])
   end
+
+  test "when Archive.org returns 429 Too Many Requests it should notify Sentry with RateLimitExceeded" do
+    api_key = create_api_key_with_webhook
+    url = 'https://example.com/'
+
+    Media.any_instance.unstub(:archive_to_archive_org)
+    Media.stubs(:get_available_archive_org_snapshot).returns({ status_ext: 'error:rate-limit', message: '429 Too Many Requests', url: url })
```

Comment:
@jayjay-w just a couple questions: 'Too Many requests' is usually an HTML response. Should we change how we are stubbing this here? And can we be sure '429 Too Many Requests' will be present in the `body['message']`?

Reply:
Thanks you! Looks good to me!